src/test/java/net/minecraftforge/debug/world/RaidEventTest.java

```diff
@@ -0,0 +1,69 @@
+package net.minecraftforge.debug.world;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.monster.PillagerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.Hand;
+import net.minecraft.world.raid.Raid.WaveMember;
+import net.minecraftforge.event.world.RaidEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("raid_event_and_more_test")
+@Mod.EventBusSubscriber
+public class RaidEventTest 
+{
+	public static final boolean ENABLE = false;
+	private static Logger LOGGER = LogManager.getLogger(RaidEventTest.class);
+	
+    public RaidEventTest()
+    {
+        if (ENABLE)
```

Comment:
@diesieben07 Again, tabs, also not that big of a deal, but simpler way would just to not use @EventBusSubscriber, but instead just register the handler gated behind a single ENABLE. Just cleaner code that way.